src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs

```diff
@@ -135,8 +135,8 @@ public AntiforgeryTokenSet GetTokens(HttpContext httpContext)
             ValidateTokens(httpContext, tokens);
         }
 
-        /// <inheritdoc />
-        public void ValidateTokens(HttpContext httpContext, AntiforgeryTokenSet antiforgeryTokenSet)
+        // Internal for testing
+        internal void ValidateTokens(HttpContext httpContext, AntiforgeryTokenSet antiforgeryTokenSet)
```

Comment:
IMO either just make it private and remove these tests. The tests are there basically for argument validation IIRC and there's a fair bit of duplication.